Sources/MetalSprockets/Core/Element.swift

```diff
@@ -1,7 +1,57 @@
 import MetalSprocketsSupport
 
+// MARK: - Element Protocol
+
+/// The fundamental building block of MetalSprockets render graphs.
+///
+/// `Element` is analogous to SwiftUI's `View` protocol. Each element describes
```

Comment:
"Each element describes a unit of GPU work" a bit of a simplification. Elements can be a lot of things: modifiers, groups, etc etc